### PR TITLE
Add support for Raspberry Pi 64 bit

### PR DESCRIPTION
**Pull request by janaperic**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,20 @@
 FROM ubuntu:20.04
 
-RUN apt-get update && apt-get install -y wget gcc-arm-linux-gnueabi gcc-arm-linux-gnueabihf git build-essential bison flex bc u-boot-tools
+RUN apt-get update && \
+	apt-get install -y \
+		wget \
+		gcc-arm-linux-gnueabi \
+		gcc-arm-linux-gnueabihf \
```
Comment (lluiscampos) [inline Dockerfile:7]: This one is not used, right? Please remove

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+		git \
+		build-essential \
+		bison \
+		flex \
+		bc \
+		u-boot-tools \
+		# to compile for 64bit ARM, particularly RPi3/4/5
```
Comment (lluiscampos) [inline Dockerfile:14]: Thank you for the formatting and the comment. I suggest to remove the unused one (see above) and to add a comment for `gcc-arm-linux-gnueabi` that it is used for BBB

File: Dockerfile [hunk 3]
```diff
@@ -1,6 +1,20 @@
 FROM ubuntu:20.04
```
Comment (lluiscampos) [inline Dockerfile:1]: Note to self,

About time to update :melting_face:

File: build-uboot-rpi64.sh [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+rpi_board=$2
+
+echo "$rpi_board: starting"
+
+# ARM 64bit build
+export CROSS_COMPILE=aarch64-linux-gnu-
+export ARCH=aarch64
+
+# Test if the toolchain is actually installed
+aarch64-linux-gnu-gcc --version
```
Comment (lluiscampos) [inline build-uboot-rpi64.sh:55]: I suggest reusing the variable to avoid typos:
```suggestion
${CROSS_COMPILE}gcc --version
```

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Thank you!

Apart from the minor nitpicks below, two high level things:
* Missing the LICENSE file update - see CI failure
* The last two commits should be squashed in one

It looks good otherwise :rocket:

**Comment by janaperic** [comment]
> Missing the LICENSE file update - see CI failure

@lluiscampos can you send me the failing build?

File: Dockerfile (hunk 2 above)
Comment (janaperic) [reply Dockerfile:12]: Removed!

File: build-uboot-rpi64.sh (hunk 4 above)
Comment (janaperic) [reply build-uboot-rpi64.sh]: Done.

File: Dockerfile (hunk 1 above)
Comment (janaperic) [reply Dockerfile]: Removed!

Comment (janaperic) [reply Dockerfile]: As it turns out, we do need gcc-arm-linux-gnueabi and gcc-arm-linux-gnueabihf. I'm bringing them back.

Comment (lluiscampos) [reply Dockerfile:7]: No, we don't need `gcc-arm-linux-gnueabihf` and it should be removed (and by removing I don't mean commented out, I mean removing the line)

File: Dockerfile (hunk 2 above)
Comment (lluiscampos) [reply Dockerfile:14]: I still think that a comment will be nice.

File: build-uboot-rpi64.sh [hunk 5]
```diff
@@ -52,7 +52,7 @@ export CROSS_COMPILE=aarch64-linux-gnu-
 export ARCH=aarch64
 
 # Test if the toolchain is actually installed
-aarch64-linux-gnu-gcc --version
+${CROSS_COMPILE}gcc --version
```
Comment (lluiscampos) [inline build-uboot-rpi64.sh:55]: This should be part of the previous commit.

This Pull Request introduces this script for the first time, hence it does not have any historical value to look at how it was first introduced and then modified the --version command to use the `CROSS_COMPILE` variable. -> This is my reasoning for whatever happened in this script should be self-contained in one commit.

**Review by lluiscampos — CHANGES_REQUESTED** [review]
@janaperic With your last force-pushed, we removed @TheYoctoJester rebase. And because you are reusing his branch, now it is a bit tricky to get it back. I recommend you do the rebase from master yourself and amend his signature.

See comments below too.

**Comment by lluiscampos** [comment]
Requesting changes for the removal of `gcc-arm-linux-gnueabihf`. The rest of the comments are suggestions.

File: build-uboot-rpi64.sh (hunk 5 above)
Comment (TheYoctoJester) [reply build-uboot-rpi64.sh:55]: Folded it into the mentioned commit.

**Comment by TheYoctoJester** [comment]
> Requesting changes for the removal of `gcc-arm-linux-gnueabihf`. The rest of the comments are suggestions.

Ok, done.

**Review by lluiscampos — APPROVED** [review]
Splendid. Thank you :rocket:

Before merging here, we need to review and merge https://github.com/mendersoftware/uboot-mender/pull/9 - otherwise we will be publishing the binaries from the wrong u-boot source.

But at lest this PR is finally ready :+1: Thank you both of you!

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**